userspace/falco/app/options.cpp

```diff
@@ -106,12 +107,18 @@ bool options::parse(int argc, char **argv, std::string &errstr)
 
 	if(m_cmdline_parsed.count("t") > 0)
 	{
+		falco_logger::log(falco_logger::level::WARNING, "The -t option is deprecated and will be removed in Falco 0.39.0. Use -o rules[].disable.rule=* -o rules[].enable.tag=<tag> instead.");
 		for(auto &tag : m_cmdline_parsed["t"].as<std::vector<std::string>>())
 		{
 			enabled_rule_tags.insert(tag);
 		}
 	}
 
+	if(disabled_rule_substrings.size() > 0)
+	{
+		falco_logger::log(falco_logger::level::WARNING, "The -d option is deprecated and will be removed in Falco 0.39.0. Use -o rules[].disable.rule=<wildcard-pattern> instead.");
```

Comment:
```suggestion
		falco_logger::log(falco_logger::level::WARNING, "The -D option is deprecated and will be removed in Falco 0.39.0. Use -o rules[].disable.rule=<wildcard-pattern> instead.");
```

The `-d` was deprecated a while ago and meant daemon mode.

Reply:
If I had a cent for every time I mixed `-d`, `-t`, `-T`, `-D`... :D good catch!

Reply:
Good thing that we deprecate them ...